src/spring-cloud/azext_spring_cloud/_buildservices_factory.py

```diff
@@ -56,14 +57,20 @@ def _get_upload_info(self):
         except AttributeError as e:
             raise AzureInternalError("Failed to get a SAS URL to upload context. Error: {}".format(e))
 
-    def _queue_build(self, relative_path=None, builder=None, target_module=None, app=None, **_):
+    def _queue_build(self, relative_path=None, builder=None, target_module=None, runtime_version=None, app=None, **_):
         subscription = get_subscription_id(self.cmd.cli_ctx)
         service_resource_id = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.AppPlatform/Spring/{}'.format(subscription, self.resource_group, self.service)
         properties = models.BuildProperties(
             builder='{}/buildservices/default/builders/{}'.format(service_resource_id, builder),
             agent_pool='{}/buildservices/default/agentPools/default'.format(service_resource_id),
             relative_path=relative_path,
             env={"BP_MAVEN_BUILT_MODULE": target_module} if target_module else None)
+        if(runtime_version == SupportedRuntimeValue.JAVA8):
```

Comment:
If your target module is `None`, the env.add will throw NPE

Reply:
OK. fixed.